brands/amenity/bank.json

```
@@ -8084,6 +8084,21 @@
       "name:en": "Bank of Kyoto"
     }
   },
+  "amenity/bank|伊予銀行": {
+    "countryCodes": ["jp"],
+    "matchNames": ["いよぎん"],
+    "tags": {
+      "amenity": "bank",
+      "brand": "伊予銀行",
+      "brand:en": "Iyo Bank",
+      "brand:ja": "伊予銀行",
+      "brand:wikidata": "2895314",
```

A Q is missing here. Thinking of it, this should have been detected by the build script... it's probably easy to add a check for it.

```suggestion
      "brand:wikidata": "Q2895314",
```

---

Yes, the build script does already detect and warn about `brand:wikidata` with incorrect format.

---

OK. I added the missing Q.